packages/__docs__/buildScripts/utils/getReactDoc.js

```diff
@@ -35,7 +35,8 @@ module.exports = function getReactDoc(source, fileName, error) {
       reactDocgen.resolver.findAllExportedComponentDefinitions,
       null,
       {
-        filename: fileName
+        filename: fileName,
+        importer: reactDocgen.importers.makeFsImporter()
```

Comment:
This new upcoming feature of `react-docgen` allows us to operate on imported types as well.